Makefile

```diff
@@ -20,13 +20,13 @@ $(error Environment variable BOLOS_SDK is not set)
 endif
 include $(BOLOS_SDK)/Makefile.defines
 
-APP_LOAD_PARAMS= --path "44'/12586'" --appFlags 0x240 $(COMMON_LOAD_PARAMS)
+APP_LOAD_PARAMS= --path "44'/12586'" --curve secp256k1 --appFlags 0x240 $(COMMON_LOAD_PARAMS)
```

Comment:
Why is it needed to add this curve? Is it just something missing? Any error seen during the tests?

Reply:
good practice: the curve(s) used shall be explicitly added in the makefile (which is checked with our guidelines workflow).